File: Python/5kyu/TheHashtagGenerator.py
def generate_hashtag(s):
    output = "#"

    for word in s.split():
        output += word.capitalize()

    return False if len(s) == 0 or len(output) > 140 else output


def generate_hashtag_kata_1(s):
    ans = "#" + str(s.title().replace(' ', ''))
    return s and not len(ans) > 140 and ans or False


result = generate_hashtag_kata_1('CodeWars is nice       !')
print(result)

File: JavaScript/5kyu/TheHashtagGenerator.js
function generateHashtag(str) {
  str = str.match(/\w+/g);

  if (!str) return false;

  var result = str
    .map((w) => w.slice(0, 1).toUpperCase() + w.slice(1).toLowerCase())
    .join("");

  return "#" + result && result.length < 140 ? result : false;
}

function generateHashtag_kata_1(str) {
  return str.length > 140 || str.length === ""
    ? false
    : "#" + str.split(" ").map(capitalize).join("");
}

const capitalize = (word) =>
  word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();

var result = generateHashtag_kata_1("Codewars   is nice");
console.log(result);
